### DISCUSSION TITLE
Image audio response reaction time

### DISCUSSION THREAD

**Discussion by ghost** [jspsych/jsPsych#2438] [Q&A]
I am really getting stuck trying to obtain the reaction time automatically from the image-audio-response plugin. I don't know what I would need to add to the example below to get an RT for each image. Currently when running this example, the RT is the same for each stimulus no matter what I say. There is even a RT recorded when I am silent. 
https://github.com/becky-gilbert/jsPsych/blob/audio-response/examples/jspsych-image-audio-response.html
I have tried looking through this documentation but I find it extremely confusing:
https://github.com/becky-gilbert/audio-response-timing/blob/master/experiments/jspsych-image-audio-response.js
The example linked above doesn't specify recording start and finish, etc. because I assume default values are implemented. I am really stuck on this and would really appreciate some help. Thank you

**Comment by jodeleeuw** [comment]
Hi @Vizitok,

My understanding is that @becky-gilbert's plugin doesn't automatically extract the RT of the spoken response. If you want that information you'll need to process the audio data generated by the plugin. This is also how the [html-audio-response plugin](https://www.jspsych.org/7.1/plugins/html-audio-response/) that is included with jsPsych works.

**Reply by ghost** [reply]
Thank you. So if I save the audio, I should be able to extract RT using the html-audio-response plugin?

**Reply by jodeleeuw** [reply]
The plugins unfortunately won't extract the RT for you. You'll have to open the recorded audio in another program to measure the delay between the start of the file and the onset of the response.
